code/modules/urist/mob/spoopymobs.dm

```diff
@@ -21,11 +21,131 @@
 	move_to_delay = 7
 	stat_attack = 1
 	ranged = 0
-	universal_speak = 1
 	var/plague = 0 //whether biting dead people spawns new zombies
 	var/regen = 0 //if true, they won't stay down, but revive after a delay
 	var/regen_delay = 900 //delay for regen revive
 
```

Comment:
Couldn't find a good place to put this. Zombies do not break tables, barricades or anything else in their way, and as such I have to rate them 0/10.

Reply:
That's not really endemic to this mob, nothing does AFAIK. Hostiles' code is pretty slapdash in general, although, the random fixes I've put in here already addressed some Big Issues it had.

Reply:
This is something I'd like to see as well, although if hostiles aren't breaking tables or barricades, then that's something that should be made into an issue

Reply:
FINALLY done. Argh.